### ISSUE TITLE
T002 lacks CN :cn: aviation data and recent years :cn:

### ISSUE THREAD

**Issue by francescolovat** [transportenergy/database#70]
Hi all,

I'm working with @khaeru at IIASA with the MESSAGEix-Transport module.

This issue is about data for China (CN). Data from [NBSC](http://www.stats.gov.cn/english/ClassificationsMethods/Definitions/200205/t20020517_72380.html) shown in the [Statistical Yearbook 2019](http://www.stats.gov.cn/tjsj/ndsj/2019/indexeh.htm) (SY2019) includes activity indicators for both passenger and freight aviation. However, these are not reflected in `T000.csv` & `T002.csv`, neither being retrieved when calling the data-processing function `historical.process()`.

I guess this is because this data is also _not_ accounted in the upstream source [OECD iLibrary (ITF Transport Statistics)](https://www.oecd-ilibrary.org/transport/data/itf-transport-statistics/passenger-transport_g2g5557f-en):
![pic3](https://user-images.githubusercontent.com/56022262/115601184-c5817580-a2dd-11eb-8f39-8ac3b630a090.png).

Can/will these two transport activity indicators be included in the iTEM dataset? Also, it appears not to be explicitly mentioned that both measures are including either _international_ or _domestic_ values (or both).

On the other hand, by _recent years_ is meant 2018, for which data is missing in the [ITF OECD iLibrary](https://www.oecd-ilibrary.org/transport/data/itf-transport-statistics/passenger-transport_g2g5557f-en) page, and it is already available from the most recent [China Statistical Yearbook 2019](http://www.stats.gov.cn/tjsj/ndsj/2019/indexeh.htm). Could also these 2018 be updated?

Finally, I've also compared SY2019 _16-20 Possession of Civil Vehicles_ (i.e. stocks) for 2015 and 2014 with [OICA](https://www.oica.net/category/vehicles-in-use/) values and they are quasi-equal (162.8 vs 162.3 million) :white_check_mark:. Thus, this means that it could also be added to `T009` and `T010` 2016-2018 values for CN, assuming that the Civil/Private distinction matches the Passenger/Commercial distinction from OICA.

**Comment by soniayeh** [comment]
You are correct. T000 and T002 are based on OECD Library, and as your screenshot suggests, include only "inland" transport (i.e. no aviation). Indeed, it would be really great if people can start contributing to new data, following the rules that we published in the documentation. For example, data from NBSC will need to be cleaned, and standardized, and also need to be given a new database ID. I would be happy to work with whoever wants to contribute.

**Comment by soniayeh** [comment]
We have been working on incorporating ATO 2023 data https://asiantransportoutlook.com/ into iTEM. The new dataset will add new data focusing on Asian countries include China Aviation, among others. (but data quality is another separate discussion) Stay tuned.
